pom.xml

```diff
@@ -21,6 +21,7 @@
         <groupId>org.apache</groupId>
         <artifactId>apache</artifactId>
         <version>31</version>
+        <relativePath />
```

Comment:
```suggestion
```

Reply:
`<relativePath />` is to fix Maven reload error in IDEA.

Reply:
got it